src/Filter/AbstractDateFilter.php

```diff
@@ -49,6 +50,17 @@ abstract class AbstractDateFilter extends Filter
 
     public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value)
     {
+        /* NEXT_MAJOR: Remove this deprecation and update the typehint */
+        if ($queryBuilder instanceof ProxyQuery) {
```

Comment:
You should invert this check (and the others)

Reply:
Yes indeed ! It's fixed, thanks.